components/style/lib.rs

```diff
@@ -87,6 +87,8 @@ extern crate smallvec;
 #[macro_use]
 extern crate style_traits;
 extern crate time;
+#[allow(unused_extern_crates)]
```

Comment:
Oops, `./mach test-unit` generates error this time(It couldn't find unicode_segmentation crate). Maybe we need to set gecko flag in test-unit.